source/extensions/filters/http/jwt_authn/jwks_async_fetcher.cc

```diff
@@ -12,7 +12,13 @@ namespace JwtAuthn {
 namespace {
 
 // Default cache expiration time in 5 minutes.
```

Comment:
Default cache expiration time in ~~5~~ 10 minutes ?

Reply:
Need to fix the documentation also. https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/jwt_authn/v3/config.proto.html?highlight=jwt#extensions-filters-http-jwt-authn-v3-remotejwks

Reply:
Thanks for catching. Will update